docs/arcaflow/workflows/schemas.md

```diff
@@ -40,8 +39,8 @@ joined with a single underscore, `_`.
 
 ### Object Names
 
-The name of an `ObjectSchema` is their `Object ID`. `ListSchema`s that 
-have an `ObjectSchema` as their item value use the name of that `ObjectSchema`.
+The name of an `ObjectSchema` is its `Object ID`. `ListSchema`s that 
+have an `ObjectSchema` as its item value uses the name of that `ObjectSchema`.

```

Comment:
I _think_ that "`ListSchema`s" is plural (i.e., it's followed by "have"), so it doesn't match "its" and "uses"; I recommend singularizing it:
> A `ListSchema` that has [...]

Reply:
i can do that